src/cclib/parser/orcaparser.py

```diff
@@ -62,6 +62,118 @@ def extract(self, inputfile, line):
         if "Program Version" in line:
             self.metadata["package_version"] = line.split()[2]
 
+        """
+================================================================================
+                                        WARNINGS
+                       Please study these warnings very carefully!
+================================================================================
+
+Warning: TCutStore was < 0. Adjusted to Thresh (uncritical)
+
+WARNING: your system is open-shell and RHF/RKS was chosen
+  ===> : WILL SWITCH to UHF/UKS
+
+
+INFO   : the flag for use of LIBINT has been found!
+
+================================================================================
+        """
+        if "WARNINGS" == line.strip():
```

Comment:
It is more Pythonic to do this the other way around, namely `line.strip() == "WARNINGS".